index.html

```diff
@@ -131,6 +131,32 @@ <h2 class="connect">LET'S CONNECT</h2>
             </ul>
             </div>
         </section>
+        <Section class="form-Container">
+            <form id="form" action="https://formspree.io/f/mwkyoojy" method="post">
+                <h2 id="contact-Me">Contact Me</h2>
+                <p class="contact-Me-Paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                <ul class="form-Ul">
+                    <li class="form-Li">
+                      <label for="name"></label>
+                      <input type="text" id="name" name="user_name" placeholder="Yeremias" required maxlength="30">
+                    </li>
+                    <li class="form-Li">
+                      <label for="mail"></label>
```

Comment:
- [ ] [OPTIONAL] Nice job done so far, but you can use the form structure without `ul` `li` tags - I think, it is not needed here, am I? 👍

Reply:
I am not sure if it can be done without the ul and li tags.  I basically followed the format of the mdn material and they used it in their example. @elmar8287  . ALso I am afraid to remove it at this point because I believe some of my CSS style is tied to the ul li classess

Reply:
> I am not sure if it can be done without the ul and li tags. I basically followed the format of the mdn material and they used it in their example. @elmar8287 . ALso I am afraid to remove it at this point because I believe some of my CSS style is tied to the ul li classess

@oshanedesign that is why I marked this comment as optional. So, you can go with your existing structure.